Logger.java

```diff
@@ -120,6 +121,15 @@ static void logException(Exception e) {
     }
   }
 
+  static String getConfigText() throws IOException {
+    String value = null;
```

Comment:
We have a lot of copies of this code, see e.g. getClient, getIntVar. Maybe make a function for reading the whole file to a string?